var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -78,17 +79,21 @@ def install(self, spec, prefix):
 
         configure(*options)
         make()
+        make("check")
```

Comment:
I recall that FFTW's self-tests are expensive.

What happens if they fail? Will the failure be silently ignored? Then the tests are kind of useless. Or will it prevent FFTW from being installed? If so -- are the tests sufficiently robust that they always pass? (That's not the case for all packages.)

In general I think that self-tests should only be run if the user explicitly requests them, unless they (1) run fairly quickly (they don't more than double the install time), and (2) are reliable (they are actually supposed to pass, and don't leave junk behind).